pkgs/development/python-modules/pypiserver/default.nix

```diff
@@ -35,6 +35,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     pip
+    setuptools
```

Comment:
I don't think we need `setuptools` in both, so can you remove it from `nativeBuildInputs`?